docs/great-houses/house-of-daos.mdx

```diff
@@ -1,104 +1,119 @@
----
-title: 🏠 House of DAOs
----
-
 <details>
 
-
 <summary>Wait, WTF are DAOs?</summary>
 
-<br />
-
-Short for “Decentralized Autonomous Organizations”, DAOs are a new type of organizations enabled by the blockchain technology. Decentralized organizations are really nothing new, what makes DAOs different from decentralized organizations is that they are constituted on the blockchain and their governance is, at least in theory, controlled by smart contracts.
-
-This allows people to start organizations with complete strangers from around the world and not worry whether one of them will run away with the money, where will the organization be formed and how will they govern its treasury. The ownership of a DAO is usually tokenized, distributed among members and it’s issuance not centrally controlled by anyone.
-
-There are multiple types of “decentralizations”. Some will say “A DAO” only implies the architectural decentralization, the technology itself. While one may have “a DAO” whose governance is centralized in a single or a few big shareholders, we do not consider those true DAOs and instead refer to them as DACs (decentralized autonomous corporations).
-
-If you ask most of us, a DAO should be both architecturally AND politically decentralized to truly be considered a DAO. For a DAO to be considered politically decentralized, it’s voting power should be sufficiently decentralized among its members. How much is sufficient? Well who the hell knows, we’ll figure it out.
-
-If you ask me, DAOs are here to enable us to build a world of cooperatively ran organizations and collectively owned platforms & infrastructure.
-
-To recap:
-
-- There is no centralized control in the DAO. There is no “CEO” and no single individual may dictate direction.
-- There isn’t a hierarchy in any traditional sense, no “managerial” & “executive” levels.
-- Members allocate resources and make decisions about the direction and initiatives through voting on proposals or practicing other decentralized decision making methods, eg. [Holacracy’s Integrative Decision Making Process](https://medium.com/org-hacking/holacracys-integrative-decision-making-process-f750d4b82abc)
-- There is a complete transparency of every transaction.
-
-</details>
-
-
-### DAO Frameworks
-
-[DAOHaus](https://daohaus.club/) - Magic internet communities for all. Share resources & get shit done. Together.
-[Aragon](https://aragon.org/) - Aragon gives internet communities unprecedented power to organize around shared values and resources.
-[DAOstack](https://daostack.io/) - An open source project advancing the technology and adoption of decentralized governance.
-[Colony](https://colony.io/) - Organizations, for the Internet. Colony makes it easy for people all over the world to build organizations together, online.
+<br>
```

Comment:
`<br>` needs to be `<br/>`

Reply:
Tbh I just copy pasted from here:
![image](https://user-images.githubusercontent.com/55857222/95649111-31d41e80-0adc-11eb-9f7e-f5e7a0b74a22.png)
and it worked the first time.. I think its only because I messed up with the </details> one?
Thank you for reviewing